test/helper/randomNumberGenerator.test.js

```diff
@@ -0,0 +1,36 @@
+const { getRandomInt } = require('../../src/helper/randomNumberGenerator');
+const { expect, assert } = require('chai');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`chai` import should occur before import of `../../src/helper/randomNumberGenerator`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5483065)